Gemfile

```diff
@@ -41,7 +41,7 @@ gem 'rack-attack'
 gem 'rack-freeze'
 
 # Database (postgres)
-gem 'pg', '~> 1.1'
+gem 'pg', '~> 1.5.9'
```

Comment:
I don't know that this actually changes anything of note in the Gemfile.lock but it seems alright I guess to put it here.